app/services/trails_service.rb

```diff
@@ -0,0 +1,24 @@
+class TrailsService
+  attr_reader :connection
+
+  def initialize
+    @connection = Faraday.new(url: "https://outdoor-data-api.herokuapp.com/api")
+  end
+
+  def trails
+    parse(connection.get do |req|
+      req.params['api_key'] = 'b9cfeeefd2bde349eedce7e901254da5' #ENV['trailapi_key']
```

Comment:
Line is too long. [85/80]<br>Missing space after #.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.